diff --git a/src/Commands/WebformSanitizeSubmissionsCommands.php b/src/Commands/WebformSanitizeSubmissionsCommands.php
index 0db7d1bff..d8aca13e2 100644
--- a/src/Commands/WebformSanitizeSubmissionsCommands.php
+++ b/src/Commands/WebformSanitizeSubmissionsCommands.php
@@ -7,7 +7,7 @@ use Drupal\Core\Database\Connection;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Extension\ModuleHandlerInterface;
 use Drush\Commands\DrushCommands;
-use Drush\Drupal\Commands\sql\SanitizePluginInterface;
+use Drush\Commands\sql\sanitize\SanitizePluginInterface;
 use Symfony\Component\Console\Input\InputInterface;
 
 /**
